docs/connectors/connector-list.mdx

```diff
@@ -0,0 +1,78 @@
+---
+title: "Connector List"
+sidebar_position: 7
```

Comment:
I find it relatively hard to find the list of connectors, especially since I'd expect this to be one of the more visited pages. Should we move higher, say after "Installing Connectors"? Or maybe make it a top-level item? 🤔

Reply:
yeah I'll pull it up the list. A top-level item feels a little strange since we'd have two connector related items.